cmd/openshift-install/main.go

```diff
@@ -18,6 +22,16 @@ var (
 )
 
 func main() {
+	cname := strings.TrimSuffix(filepath.Base(os.Args[0]), ".exec")
```

Comment:
Why is the `.exec` needed? Do we need to trim `.exe` for Windows?

Reply:
Ahh, missed that I had `exec` , my brain was reading it as `exe`.  This should definitely be `exe` 😅.

Reply:
@wking fixed with https://github.com/openshift/installer/pull/919/commits/10b4cc733aa290fc5f9749cfcace4850d2685a05

PTAL